webapp/src/components/sidebar/sidebar.tsx

```diff
@@ -111,10 +101,8 @@ const Sidebar = React.memo((props: Props) => {
                 activeBoardId={props.activeBoardId}
             />
 
-            <SidebarSettingsMenu
-                setWhiteLogo={(newWhiteLogo: boolean) => setWhiteLogo(newWhiteLogo)}
-                activeTheme={getActiveThemeName()}
-            />
+            {(window as any).isFocalboardPlugin !== true &&
```

Comment:
nit: this reads a bit weird, maybe is better to use something like:
```suggestion
            {!(window as any).isFocalboardPlugin &&
```